dev-python/latexcodec/latexcodec-2.0.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_6 python3_7 python3_8 )
```

Comment:
You can use the nice python3_{6,7,8} syntax here, by the way.

Reply:
Switched to nicer syntax :)